src/styles/blocks/card.scss

```diff
@@ -0,0 +1,128 @@
+.container {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the value 'repeat(1, 200px)' can be simplified to '200px' since repeating once is redundant.